### PR TITLE
Followup edits to the scaling and performance networking info

### PR DESCRIPTION
**Pull request by bfallonf**
As per #4171 

Some minor rewording, but @jeremyeder I had a question. I'll put it inline.

Thanks for the work on this!

### CODE REVIEW

File: scaling_performance/network_optimization.adoc [hunk 1]
```diff
@@ -114,18 +114,19 @@ in latency tests.
...
 
-Restrictions for resizing networks are document here:
+Restrictions for resizing networks are document in the
 xref:../install_config/configuring_sdn.adoc#configuring-the-pod-network-on-masters[Configuring
 SDN documentation].
 
-If you would like to plan for a larger environment, here are some example values to consider adding to the `[OSE3:vars]` section in your
-Ansible inventory file:
+To plan for a larger environment, the following are suggested values to consider
+adding to the `[OSE3:vars]` section in your Ansible inventory file:
```
Comment (bfallonf) [inline scaling_performance/network_optimization.adoc]: @jeremyeder did you intentionally erase the other values for the inventory file as well? Also, is this supposed to happen at install?

**Comment by ahardin-rh** [comment]
@bfallonf @jeremyeder I see this is still a WIP. Once this merges, I can knock out the work for: https://trello.com/c/eI6qJFRU/407-docs-break-routing-and-network-performance-into-different-sections-3

**Comment by bfallonf** [comment]
Hmm forgot about this. I guess @jeremyeder let me know what you think about my question above and @ahardin-rh or I can get this merged.

**Comment by ahardin-rh** [comment]
@jeremyeder bump

File: scaling_performance/network_optimization.adoc (hunk 1 above)
Comment (jeremyeder) [inline scaling_performance/network_optimization.adoc]: Which other values are you referring to @bfallonf ?

Comment (bfallonf) [inline scaling_performance/network_optimization.adoc]: Sorry @jeremyeder  I meant in your initial PR:

https://github.com/openshift/openshift-docs/pull/4171/files#diff-63cc6ae3cf2d348826dbb2e7f6cb0b35R132

File: scaling_performance/network_optimization.adoc [hunk 2]
```diff
@@ -134,4 +135,7 @@ osm_cluster_network_cidr=10.128.0.0/10
 
 This will allow for 8192 nodes, each with 510 usable IPs.
 
-Please refer to the supportability limits in the {product-title} documentation for node/pod limits for the version of software you are installing.
+Please refer to the supportability limits in the {product-title} documentation
```
Comment (ahardin-rh) [inline scaling_performance/network_optimization.adoc]: s/Please refer/Refer

**Comment by bfallonf** [comment]
@jeremyeder We all forgot about this one and the bot reminded us. I'll send you an email in case you're not up on github notifications, but we can probably close this.

**Comment by bfallonf** [comment]
Received ack over email. Merging.

**Pull request merged**